Apigee/Mint/Organization.php

```diff
@@ -699,12 +702,10 @@ public function getPrepaidBalanceReport($month, $year, $developer_id, $currency_
                 'billingYear' => $year
             );
 
-            $url = '/mint/organizations/' . rawurlencode($this->config->orgName) . '/prepaid-balance-reports';
+            $url = rawurlencode($this->config->orgName) . '/prepaid-balance-reports';
             $content_type = 'application/json; charset=utf-8';
             $accept_type = 'application/octet-stream; charset=utf-8';
-            $this->setBaseUrl($url);
-            $this->post(null, $data, $content_type, $accept_type);
-            $this->restoreBaseUrl();
+            $this->post($url, $data, $content_type, $accept_type);
```

Comment:
First parameter of the post() call is `null` previously, I'm not sure about the change.

Reply:
This is connected with change above, before `$base_url` was replaced with whole request path:
```
$url = '/mint/organizations/' . rawurlencode($this->config->orgName) . '/prepaid-balance-reports';
```
and set as base URL, that's why first parameter was null, because whole base path was set there
Now part of url needed after `$base_path` is passed as parameter of post() method